Design-System/styles/utils.css

```diff
@@ -0,0 +1,69 @@
+/* *******************Utility classes for cart********************* */
+body {
+    background-color: #eee;
+}
+.address_header {
+    display: flex;
+    flex-direction: row;
```

Comment:
this is redundant code, displat flex has row direction by default, no need to mention it

Reply:
Ok Done